docker-compose.cli.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.7'
+
+services:
+    composer:
+        image: composer
+        working_dir: /application
```

Comment:
Composer already have `/app` as working directory. Any special reason for using a new one?

Reply:
Nops, the working_dir for this image was '/', I'll change this after merge because I have other one PR based on this

Reply:
https://github.com/kdti/kdti-server/pull/117

Reply:
```bash
$ docker run --rm -it composer bash                                                                             
bash-4.4# pwd
/app
bash-4.4#
```

Reply:
See: https://github.com/composer/docker/blob/master/1.9/Dockerfile#L60